daml-foundations/daml-tools/da-hs-daml-cli/DA/Cli/Damlc.hs

```diff
@@ -208,7 +222,152 @@ execCompile inputFile outputFile opts = withProjectRoot $ \relativize -> do
 
 newtype DumpPom = DumpPom{unDumpPom :: Bool}
 
-execPackage :: FilePath -- ^ input file
+execPackageNew :: Int -> Maybe FilePath -> IO ()
+execPackageNew numProcessors mbOutFile =
+    withProjectRoot $ \_relativize -> do
+        project <- readProjectConfig $ ProjectPath "."
+        case parseProjectConfig project of
+            Left err -> throwIO err
+            Right (name, main, exposedModules, version, dependencies) -> do
```

Comment:
Maybe make an ADT for this instead of using a 5-tuple?

Reply:
done.